components/main/MainContent.vue

```diff
@@ -63,3 +68,19 @@ const containerClass = computed(() => {
     </div>
   </div>
 </template>
+
+<style>
+  .beta {
+    width: 56.16px;
+    height: 35.16px;
+    padding: 5px 6px;
+    border-radius: 8px;
+    border: 2px solid var(--c-primary);
+
+    color: var(--c-primary);
+    font-size: 18px;
+    font-weight: 700;
+    line-height: 21px;
+    letter-spacing: 0em;
+  }
+</style>
```

Comment:
Also, Is there a reason we are switching to styles like this as opposed to sticking with the unoCSS paradigm? `border-r-8` as opposed to a specific.  Don't get me wrong, I am all for styles over framework, but I would love to better understand if there is a need to mix it here?

Reply:
I had not realized there was a border rule. I have not dived too deeply into the rules beyond positioning rules. I'll go through rn and reply back to this thread

Reply:
I was only able to find `border-rounded`, which produces a less fun shape - 
<img width="195" alt="image" src="https://github.com/MozillaSocial/elk/assets/14023085/b2d29828-4734-43e6-8f74-b8ed38003709">


I'm preferenced to keep our own border-rule. I think it makes a big difference not here but on the Footer's label

Reply:
I was able to figure this out and will have a PR soon!